vars/runInsidePod.groovy

```diff
@@ -0,0 +1,77 @@
+import com.sap.piper.ConfigurationHelper
```

Comment:
do we really need a separate step here? Wouldn't this rather be a helper function in the kubernetes docker execution step

Reply:
No, because we want to offer the possibility to create an pod outside the step, e.g. for a stage already containing all containers needed within the stage:
https://github.wdf.sap.corp/MA/cloud-s4-sdk-pipeline-lib/blob/feature/k8s-stage/vars/runAsStage.groovy#L44